tensorboard/compat/tensorflow_stub/io/gfile.py

```diff
@@ -366,6 +445,15 @@ def read(self, n=None):
 
         return result
 
+    def write(self, file_content):
+        if not self.write_mode:
+            raise errors.OpError(None, None, "File not opened in write mode")
+
+        # add to temp file, but wait for flush to write to final filesystem
+        if self.write_temp is None:
+            self.write_temp = tempfile.TemporaryFile("w+b")
+        self.write_temp.write(compat.as_bytes(file_content))
```

Comment:
Again I'd like to be strict here - we should open the tempfile in binary mode to match self.binary_mode, and avoid doing the auto-conversion.